packages/core/src/shared/date/index.ts

```
@@ -39,5 +39,7 @@ export type {
   TimeSegmentObj,
   TimeSegmentPart,
 } from './types'
+export { type CellKeydownOptions, type CellNavigationHost, createCellFocusNavigation } from './cellFocusNavigation'
```

ESLint

[error] Expected "./cellFocusNavigation" to come before "./types".

(perfectionist/sort-exports)